android/README.md

```diff
@@ -64,12 +64,14 @@ export SENTRY_PROJECT=keyman-android
 To validate your configuration, from the `android/` folder run `sentry-cli info`.
  
 ### Compiling From Command Line
-1. Launch a command prompt
-2. Change to one of these directories depending on what you want to compile:
-    * For compiling KMEA and KMAPro, cd to the directory **keyman/android**
-    * For compiling only KMAPro, cd to the directory **keyman/android/KMAPro**
-3. `./build.sh -debug`
-4. The APK will be found in **KMAPro/kMAPro/build/outputs/apk/debug/kMAPro-debug.apk**
+1. Launch a command prompt and cd to the directory **keyman/android**
+2. Run the top level build script `./build.sh -debug -download-resources` which will:
+    * Compile KMEA (and its KMW dependency)
+    * Download default keyboard and dictionary resources
+    * Compile KMAPro
+    * Note: If you've already downloaded resources before, you can skip the `-download-resources` flag.
```

Comment:
Can we set this up so if the resources are missing it will attempt to download (but it won't do an update unless the flag is present)?

Reply:
> but it won't do an update unless the flag is present

Could you describe the scenario? I'm unclear what you mean by "update".

Reply:
For a local developer, if they don't pass `-download-resources`, then the packages that are present won't be updated even if a newer version is available (download will be skipped). However, if the package is missing, it will be downloaded.

For CI and test, the latest package should *always* be downloaded (and given that the files should not be present given we do a clean build each time, this means that the `-download-resources` flag shouldn't actually be needed there, but it doesn't hurt to provide it).